### PR TITLE
Proxy Truncation Code + Logging Statments

### PR DESCRIPTION
**Pull request by joannak-vmware**
Changes to merge into MONIT-23859 PR#668

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java [hunk 1]
```diff
@@ -244,12 +247,16 @@ void updateConfiguration() {
     try {
       AgentConfiguration config = checkin();
       if (config != null) {
-        logger.info("----> getShutOffAgents: "+config.getShutOffAgents());
-        logger.info("---->  isTruncateQueue: "+config.isTruncateQueue());
+        logger.info("Server configuration getShutOffAgents: " + config.getShutOffAgents());
```
Comment (laullon) [inline proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java:250]: better in debug

File: proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java [hunk 2]
```diff
@@ -244,12 +247,16 @@ void updateConfiguration() {
...
+        logger.info("Server configuration isTruncateQueue: " + config.isTruncateQueue());
         if (config.getShutOffAgents()) {
           logger.severe(firstNonNull(config.getShutOffMessage(),
               "Shutting down: Server side flag indicating proxy has to shut down."));
           shutdownHook.run();
+        } else if (config.isTruncateQueue()) {
+          logger.info(
```
Comment (laullon) [inline proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java:257]: severe

File: proxy/src/main/java/com/wavefront/agent/handlers/SenderTaskFactoryImpl.java [hunk 3]
```diff
@@ -146,7 +146,7 @@ public Long value() {
       senderTask.start();
     }
     if (queueingFactory != null) {
-      log.info("---->"+handlerKey);
+      log.info("Sender tasks handlerKey:"+handlerKey);
```
Comment (laullon) [inline proxy/src/main/java/com/wavefront/agent/handlers/SenderTaskFactoryImpl.java:149]: no need for this

**Review by laullon — APPROVED** [review]
minor comments, LGTM

File: proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java [hunk 4]
```diff
@@ -25,8 +25,8 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Consumer;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```
Comment (joannak-vmware) [inline proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java:29]: Changed to using log4j for debug logs

**Pull request merged**